cmd/wallet-web/src/store/modules/options.js

```diff
@@ -167,17 +116,19 @@ export default {
         agentOpts.walletMediatorURL = 'https://localhost:10093';
         agentOpts.hubAuthURL = 'https://localhost:8044';
 
-        profileOpts = {
+        Object.assign(profileOpts, {
           config: {
             storageType: defaultAgentStartupOpts.storageType,
             kmsType: defaultAgentStartupOpts.kmsType,
             localKMSScret: defaultAgentStartupOpts.localKMSPassphrase,
           },
           bootstrap: {
-            user,
-            tokenExpiry: '10',
+            data: {
+              user,
+              tokenExpiry: '10',
```

Comment:
add a TODO to make it configurable

Reply:
Filed https://github.com/trustbloc/wallet/issues/1791